wb-codemod/transforms/migrate-color-to-tokens.ts

```diff
@@ -0,0 +1,136 @@
+import {
+    API,
+    FileInfo,
+    ImportSpecifier,
+    ModuleSpecifier,
+    Options,
+} from "jscodeshift";
+
+// From
+const SOURCE_IMPORT_DECLARATION = "@khanacademy/wonder-blocks-color";
+const SOURCE_SPECIFIER = "Color";
+// To
+const TARGET_IMPORT_DECLARATION = "@khanacademy/wonder-blocks-tokens";
+const TARGET_SPECIFIER = "color";
+
+/**
+ * Transforms: import Color from "@khanacademy/wonder-blocks-color" const
+ * bgColor = Color.blue;
+ *
+ * to: import {color} from "@khanacademy/wonder-blocks-tokens" const bgColor =
+ * color.blue;
+ *
+ * NOTE: This transform is similar to migrate-spacing-to-tokens.ts, but it
+ * differs in the way it replaces the usage of the default specifier. Spacing
+ * only exports a default specifier, while Color exports both a default and
+ * named specifier(s) like `mix` and `fade`.
+ */
+export default function transform(file: FileInfo, api: API, options: Options) {
+    const j = api.jscodeshift;
+    const root = j(file.source);
+
+    // Use the default specifier as the source specifier.
+    let sourceSpecifier = SOURCE_SPECIFIER;
+
+    // Step 1: Verify if the import exists
+    const sourceImport = root.find(j.ImportDeclaration, {
+        source: {value: SOURCE_IMPORT_DECLARATION},
+    });
+
+    // If the import doesn't exist, we don't need to do anything.
+    if (sourceImport.size() === 0) {
+        return;
+    }
```

Comment:
Does `root.find()` return a `Set`?

Reply:
It returns a `Collection`, which is an interface defined in jsCodeshift: https://github.com/facebook/jscodeshift/wiki/jscodeshift-Documentation#collections